<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>38</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>38</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="16">
      <widget class="PyDMPushButton" name="JogDown">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>↓</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.VAL</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>-1</string>
       </property>
       <property name="relativeChange" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="25">
      <widget class="PyDMPushButton" name="PyDMPushButton_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Kill</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_KILL_CMD.PROC</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="26">
      <widget class="PyDMPushButton" name="PyDMPushButton_enable">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_ENA_CMD.PROC</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="28">
      <widget class="PyDMRelatedDisplayButton" name="PyDMRelatedDisplayButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>details...</string>
       </property>
       <property name="showIcon" stdset="0">
        <bool>false</bool>
       </property>
       <property name="displayFilename" stdset="0">
        <string>expert_motor.ui</string>
       </property>
       <property name="macros" stdset="0">
        <stringlist/>
       </property>
       <property name="openInNewWindow" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PyDMLabel" name="PyDMLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.DESC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="20">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="19">
      <widget class="PyDMByteIndicator" name="PyDMByteIndicator_3">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>15</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_PLIM_STS</string>
       </property>
       <property name="onColor" stdset="0">
        <color>
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="offColor" stdset="0">
        <color>
         <red>85</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="showLabels" stdset="0">
        <bool>false</bool>
       </property>
       <property name="circles" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="PyDMLabel" name="PyDMLabel_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="precision" stdset="0">
        <number>0</number>
       </property>
       <property name="showUnits" stdset="0">
        <bool>true</bool>
       </property>
       <property name="precisionFromPV" stdset="0">
        <bool>true</bool>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.RBV</string>
       </property>
      </widget>
     </item>
     <item row="0" column="18">
      <widget class="PyDMPushButton" name="JogUp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>↑</string>
       </property>
       <property name="rules" stdset="0">
        <string>[]</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.VAL</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>1</string>
       </property>
       <property name="relativeChange" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="15">
      <widget class="PyDMByteIndicator" name="PyDMByteIndicator_2">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>15</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_MLIM_STS</string>
       </property>
       <property name="onColor" stdset="0">
        <color>
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="offColor" stdset="0">
        <color>
         <red>85</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="showLabels" stdset="0">
        <bool>false</bool>
       </property>
       <property name="circles" stdset="0">
        <bool>false</bool>
       </property>
       <property name="numBits" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="PyDMLineEdit" name="PyDMLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="precision" stdset="0">
        <number>0</number>
       </property>
       <property name="showUnits" stdset="0">
        <bool>false</bool>
       </property>
       <property name="precisionFromPV" stdset="0">
        <bool>true</bool>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.VAL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="22">
      <widget class="PyDMLabel" name="PyDMLabel_3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_AXIS_STS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="27">
      <widget class="PyDMPushButton" name="PyDMPushButton_3">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_HOME_CMD.PROC</string>
       </property>
       <property name="showConfirmDialog" stdset="0">
        <bool>true</bool>
       </property>
       <property name="pressValue" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QDoubleSpinBox" name="JogValue">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="23">
      <widget class="PyDMByteIndicator" name="PyDMByteIndicator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.MOVN</string>
       </property>
       <property name="showLabels" stdset="0">
        <bool>false</bool>
       </property>
       <property name="circles" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="24">
      <widget class="PyDMPushButton" name="PyDMPushButton">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: red;</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}.STOP</string>
       </property>
       <property name="pressValue" stdset="0">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="21">
      <widget class="PyDMLabel" name="PyDMLabel_4">
       <property name="toolTip">
        <string/>
       </property>
       <property name="channel" stdset="0">
        <string>ca://${MOTOR}_HOMING_STS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMByteIndicator</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.byte</header>
  </customwidget>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
  <customwidget>
   <class>PyDMRelatedDisplayButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.related_display_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>JogValueChange(double)</slot>
 </slots>
</ui>
